### PR TITLE
fix: add input variables description and bump SaaS DA version from v1.0.0 to v1.3.6

### PR DESCRIPTION
**Pull request by akocbek**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: ibm_catalog.json [hunk 1]
```diff
@@ -258,12 +258,14 @@
...
 							"type": "array",
 							"default_value": [],
+							"description": "The list of email addresses to notify when Security and Compliance Center triggers an event.",
 							"required": false
 						},
 						{
 							"key": "scc_service_plan",
 							"type": "string",
 							"default_value": "security-compliance-center-standard-plan",
+							"description": "The service/pricing plan to use when provisioning a new Security Compliance Center instance. Possible values: `security-compliance-center-standard-plan`, `security-compliance-center-trial-plan`.",
```
Comment (ocofaigh) [inline ibm_catalog.json]: can you do the same as core security services? Use same description and add a dropdown box: https://github.com/terraform-ibm-modules/stack-ibm-core-security-services/blob/55df0ab93205aac3cf51ee08c98fba9af0536d46/ibm_catalog.json#L246-L260

File: ibm_catalog.json [hunk 2]
```diff
@@ -258,12 +258,14 @@
 							"key": "en_email_list",
 							"type": "array",
 							"default_value": [],
+							"description": "The list of email addresses to notify when Security and Compliance Center triggers an event.",
```
Comment (ocofaigh) [inline ibm_catalog.json]: Its not just SCC events, its secrets manager too. Use the same description as core security services does

**Review by ocofaigh — DISMISSED** [review]

**Comment by ocofaigh** [comment]
@akocbek For the release notes I'll need to know what version of the Watson SaaS DA we are upgrading to from. Once I know that I can generate a summary of changes for the release notes

**Comment by akocbek** [comment]
@ocofaigh from `v1.0.0` to `v1.3.6`

**Comment by ocofaigh** [comment]
@akocbek FYI the pipeline failed with this issue so I re-opened it -> https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/issues/112

**Comment by ocofaigh** [comment]
```
    tests.go:252: 
        	Error Trace:	/go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.35.4/testprojects/tests.go:252
        	            				/go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.35.4/testprojects/tests.go:368
        	            				/go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.35.4/testprojects/tests.go:463
        	            				/usr/local/go/src/runtime/asm_amd64.s:1695
        	Error:      	Received unexpected error:
        	            	Get "https://projects.api.cloud.ibm.com/v1/projects/b99dd089-1c4e-4060-a40a-7b89ec607073/configs/7bee6e47-b5dc-4de3-9abf-d07ebe3602cc": unexpected EOF
        	Test:       	TestProjectsFullTest
    tests.go:369: Error deploying configuration 4 - Security Service - Event Notifications: Get "https://projects.api.cloud.ibm.com/v1/projects/b99dd089-1c4e-4060-a40a-7b89ec607073/configs/7bee6e47-b5dc-4de3-9abf-d07ebe3602cc": unexpected EOF
    tests.go:414: [PROJECTS] Error deploying configurations, terminating deployment.
    tests.go:531: 
        	Error Trace:	/go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.35.4/testprojects/tests.go:531
        	            				/__w/stack-ibm-genai/stack-ibm-genai/tests/pr_test.go:25
        	Error:      	Should be empty, but was [Get "https://projects.api.cloud.ibm.com/v1/projects/b99dd089-1c4e-4060-a40a-7b89ec607073/configs/7bee6e47-b5dc-4de3-9abf-d07ebe3602cc": unexpected EOF]
        	Test:       	TestProjectsFullTest
    tests.go:534: Get "https://projects.api.cloud.ibm.com/v1/projects/b99dd089-1c4e-4060-a40a-7b89ec607073/configs/7bee6e47-b5dc-4de3-9abf-d07ebe3602cc": unexpected EOF
```

Unrelated to changes in this PR, so re-running..

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**